code/modules/vending/vending.dm

```diff
@@ -1596,7 +1596,9 @@
 	"Prepared in moments!",
 	"I'm a chef who works 24 hours a day!")
 	var/sharpen = FALSE
-
+	create_products()
+		..()
+		product_list += new/datum/data/vending_product(/obj/item/reagent_containers/food/snacks/pizza/random/vendor, 100)
```

Comment:
Still got `100` in it.

Reply:
Yeah that's gonna be totally refactored, and we're in draft now pending deciding how much randomness

Reply:
Now has 10!